le_utils/resources/languagelookup.json

```diff
@@ -819,7 +819,7 @@
     "native_name":"Español, Mexico"
   },
   "es-NI":{
-    "name":"Spanish, Nicaraguan",
+    "name":"Spanish, Nicaragua",
```

Comment:
For consistency with other Spanish -- use `Spanish, Country` instead of adjective.